frame/staking/src/slashing.rs

```diff
@@ -239,9 +239,9 @@ pub(crate) fn compute_slash<T: Config>(
 		return None
 	}
 
-	let (prior_slash_p, _era_slash) =
+	let prior_slash_p =
 		<Pallet<T> as Store>::ValidatorSlashInEra::get(&params.slash_era, params.stash)
-			.unwrap_or((Perbill::zero(), Zero::zero()));
+			.map_or(Zero::zero(), |(p, _)| p);
```

Comment:
maybe best to actually have a full name for this, `prior_slash_p = p` is kinda cryptic.